test/cartesian/Area.spec.tsx

```diff
@@ -892,262 +892,3 @@ describe('getBaseValue', () => {
     });
   });
 });
-
-describe('getComposedData', () => {
```

Comment:
The function and the logic is still there. Not in a static method but in a function. Can we keep the tests?

Reply:
Yeah I can do the same for the compute function I guess